### PR TITLE
[Core][Fixture] Added tax category support in shipping method fixture

### PR DESCRIPTION
**Pull request by SebLours**
| Q               | A
| --------------- | -----
| Branch?         | 1.1
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
👍

**Comment by SebLours** [comment]
@lchrusciel done, is it correct ?
https://github.com/Sylius/Sylius/pull/9692/files#diff-5b32d7a967e23e6a86812caf1d4699a3

File: src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php [hunk 1]
```diff
@@ -70,19 +77,22 @@ class ShippingMethodExampleFactory extends AbstractExampleFactory implements Exa
...
+     * @param TaxCategoryRepository $taxCategoryRepository
      */
     public function __construct(
         FactoryInterface $shippingMethodFactory,
         RepositoryInterface $zoneRepository,
         RepositoryInterface $shippingCategoryRepository,
         RepositoryInterface $localeRepository,
-        ChannelRepositoryInterface $channelRepository
+        ChannelRepositoryInterface $channelRepository,
+        TaxCategoryRepository $taxCategoryRepository
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php]: It should be an interface here - `TaxCategoryRepositoryInterface`.

**Comment by pamil** [comment]
Since it's adding an argument to the constructor (and it's labeled as "Feature" correctly), I've changed the base branch from `1.1` to `master`.

**Comment by Zales0123** [comment]
Hello @SebLours, what is the status of this PR? Would you be able to fix @pamil comment and rebase your branch with newest `master`? Thanks!

**Comment by SebLours** [comment]
@Zales0123 @pamil Done
Could you review my changes?

File: src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php [hunk 2]
```diff
@@ -55,13 +60,15 @@ public function __construct(
         RepositoryInterface $zoneRepository,
         RepositoryInterface $shippingCategoryRepository,
         RepositoryInterface $localeRepository,
-        ChannelRepositoryInterface $channelRepository
+        ChannelRepositoryInterface $channelRepository,
+        TaxCategoryRepositoryInterface $taxCategoryRepository
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php]: BC break :( We need to support default null value here as well.

**Comment by lchrusciel** [comment]
Closing in favour of #9962.

Sebastian, thanks a lot for your contribution, but we have spotted the same problem previously on the other PR, therefore it is ready to merge right now. Sorry, for that :(

**Pull request closed without merging**